modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -374,6 +397,7 @@ class ControlUtilityError(RemoteCommandError):
     """
     Error is raised when control utility failed.
     """
+
```

Comment:
Why do you make this change? Do we have a check for that?

Reply:
I think, linter  force me to do this change. If you talk about '\n' symbol

Reply:
The question is about this change!

Reply:
Added ability to specify a username and password for control_utility,
Added ability to specify plugins in configuration